### PR TITLE
Project3

### PR DESCRIPTION
**Pull request by ONJoseph**
- add styling finishing touches
- add the footer to index.html file
- add leaderboard screenshot
- update README.md file

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -42,38 +51,88 @@ input {
...
 }
+
+footer {
+  width: 100%;
+  position: fixed;
+  padding-bottom: 2rem;
+  bottom: 0;
+  text-align: center;
+  font-size: 0.8rem;
+}
```
Comment (ShahierNashaat) [inline src/styles.css]: - [ ] Kindly remove the position: fixed from the footer as the footer goes over the leaderboard scores.
Screenshot from 2022-05-12 23-37-33

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @ONJoseph , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles.css (hunk 1 above)
Comment (ONJoseph) [reply src/styles.css]: Hi @ShahierNashaat, Thanks a lot for the kind review! Suggested change has been implemented.

**Review by smunozmo — APPROVED** [review]
Hi @ONJoseph, 👋 

## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**